src/audio/volume.c

```diff
@@ -465,42 +465,51 @@ static int volume_copy(struct comp_dev *dev)
 	struct comp_data *cd = comp_get_drvdata(dev);
 	struct comp_buffer *sink;
 	struct comp_buffer *source;
+	uint32_t frames;
+	uint32_t in_bytes;
+	uint32_t out_bytes;
```

Comment:
Best to align with source/sink naming convention for in/out bytes

Reply:
Fixed.